src/Symfony/Component/Cache/CacheItem.php

```diff
@@ -20,16 +20,13 @@
  */
 final class CacheItem implements CacheItemInterface
 {
-    /**
-     * @internal
-     */
-    const CAST_PREFIX = "\0Symfony\Component\Cache\CacheItem\0";
-
-    private $key;
-    private $value;
-    private $isHit;
-    private $expiry;
-    private $defaultLifetime;
+    protected $key;
```

Comment:
Since the class is final, changing to protected here has to bc-promise-related impact.